js/app/src/components/UploadButton/UploadButton.svelte

```diff
@@ -6,13 +6,13 @@
 	import { blobToBase64 } from "@gradio/upload";
 	import { _ } from "svelte-i18n";
 
-	export let elem_id: string = "";
-	export let elem_classes: Array<string> = [];
-	export let visible: boolean = true;
+	export let elem_id = "";
+	export let elem_classes: string[] = [];
+	export let visible = true;
 	export let label: string;
 	export let value: null | FileData;
 	export let file_count: string;
-	export let file_types: Array<string> = ["file"];
+	export let file_types: string[] = [];
```

Comment:
this was the actual problem, when we pass this down it turns into `file/*` but that is not valid. The browser doesn't seem to find but the only valid options are `image/*`, `audio/*` `video/*` or an actual extension starting with `.`. More here: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept#unique_file_type_specifiers

Removing this (defaulting to nothing) fixed it. This does now need a changelog as it isn't just a test thing but this will result in the same behaviour, any file can be uploaded.